Flow.Launcher/PriorityChangeWindow.xaml.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Flow.Launcher.Core.Plugin;
+using Flow.Launcher.Core.Resource;
+using Flow.Launcher.Infrastructure.UserSettings;
+using Flow.Launcher.Plugin;
+using Flow.Launcher.ViewModel;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Shapes;
+
+namespace Flow.Launcher
+{
+    /// <summary>
+    /// PriorityChangeWindow.xaml 的交互逻辑
```

Comment:
lets update this auto generated msg to english so it's consistent throughout the project plz

Reply:
ok